packages/builder/src/components/database/ModelDataTable/modals/CreateEditRecord.svelte

```diff
@@ -27,14 +23,25 @@
   function closed() {
     onClosed()
   }
-
+  
+  const isSelect = meta =>
+    meta.type === "string" 
```

Comment:
Could we possibly just create a `select` or `choice` type for fields? Might be handier